source/ed/data.cpp

```diff
@@ -218,6 +218,43 @@ void Data::shift_stop_times() {
     }
 }
 
+bool compare(const std::pair<idx_t, size_t>& p1, const std::pair<idx_t, size_t>& p2){
+    return p1.second < p2.second;
+}
+
+void Data::build_route_destination(){
+    // <Route.idx, std::map<StopArea.idx, Count>>
+    std::unordered_map<idx_t, std::map<idx_t, size_t>> destinations;
```

Comment:
I don't like using idx_t, you can just directly use the pointers, like that, you are shure to not use an idx on the bad object:
```c++
std::unordered_map<Route*, std::unordered_map<StopArea*, size_t>> destinations;
```